src/main/java/io/vertx/proton/impl/ProtonConnectionImpl.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 import java.util.UUID;
 
+import io.vertx.proton.*;
```

Comment:
I didn't know that. A little annoying given not everyone uses IDEA and so theirs might auto reverse that :)